### PR TITLE
[FSDE-3514] Added size support to floating icon buttons

### PR DESCRIPTION
**Pull request by mattnickles**
https://ctl.atlassian.net/browse/FSDE-3514

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-157/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://390f187e422161e0719f6dd7766999f079626a91-chi-pr.enterprise.ctl.io

**Comment by mattnickles** [comment]
@janibolkvadze @jllr

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-157/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://27384def74d057f77e200b5009047f60a9e17654-chi-pr.enterprise.ctl.io

**Review by dani-cl-madrid — APPROVED** [review]

File: src/website/views/components/button/_examples.pug [hunk 1]
```diff
@@ -519,19 +541,10 @@ h2 Examples
...
-                chi-icon(icon='atom')
-      .chi-col.-w--12.-w-md--6
-        .-p--3.-bg--black.-d--flex(style="justify-content: center;")
-          each type, val in {Secondary:'secondary', Light:'light'}
-            div.-pr--2
-              chi-button(color=type, type='float', alternative-text="Button action")
-                chi-icon(icon='atom')
+    .-p--3.-d--flex
+      div.-pr--2
+        chi-button(color='primary' type='float' size='xl' alternative-text='Chat with Support' data-tooltip='Chat with Support')
```
Comment (jllr) [inline src/website/views/components/button/_examples.pug:546]: Shouldn't floating icon btns in Lumen and Centurylink adhere to 48px instead of 56px too?

Comment (mattnickles) [reply src/website/views/components/button/_examples.pug:546]: @jllr Lumen and CenturyLink dot com use cases require larger sizes

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-157/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://84b11b4a236460ca8bf3084cedcb222d2b668a17-chi-pr.enterprise.ctl.io

**Pull request merged**